email/sender.go

```diff
@@ -32,6 +32,9 @@ var (
 	// comes from the PORTAL_DOMAIN environment variable, preceded by the
 	// appropriate schema.
 	PortalAddress = "https://siasky.net"
+	// PortalAddressAccounts defines the URI where we can access the accounts
+	// sub-site.
+	PortalAddressAccounts = "https://account.siasky.net"
```

Comment:
Is there a reason this and `PortalAddress` aren't just loaded from the `.env` file?

Reply:
They probably should, otherwise community portals are going to be sending emails with `siasky.net` links in them instead of their own links.

Reply:
They are. These are just initial values. It seems that they are confusing, though. I'll remove them.

Reply:
Actually, I'm going to keep them. The reason for that is that they won't be initialised during testing unless I do it manually which is not great.

We don't need to worry about custom portals - if the `PORTAL_DOMAIN` env var which defines those is not set the service will panic on start with an appropriate error message.

Reply:
If these are only used in testing I would move them to a test file to make it more clear that these aren't duplicate production constants. 

Or use a build variable :-) 

Fine as a follow up.

Reply:
Build variable sounds great. Will do.